openpype/plugins/publish/extract_otio_audio_tracks.py

```diff
@@ -1,11 +1,18 @@
 import os
+import tempfile
```

Comment:
Has this sneaked in?

Reply:
In cleaning up the imports yeah I've rearranged them. Is that a problem?